content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -26,46 +26,48 @@ Here's a summary of the steps involved:
 
 ### 1. Install Microk8s
 
-Single-command install MicroK8s snap:
+- Install MicroK8s with Snap by running the following command:
 
 ```
 sudo snap install microk8s --classic
 ```
-Confirm install:
+
+- Verify that MicroK8s is running with the command:
+
 ```
 microk8s.status --wait-ready
 ```
 
-### 2. Enable Microk8s services:
+### 2. Enable Microk8s services
+
+- Enable common services on your Microk8s deployment:
 
-Enable common services:
 ```
 microk8s.enable dns dashboard storage
 ```
-If you would like to enable GPU passthrough (optional), run: `microk8s.enable gpu`
 
-### 3. Enable Kubeflow:
+- Optional: to enable GPU support (available only for NVIDIA GPU hardware), run: `microk8s.enable gpu`
 
-Run the following command to enable Kubeflow:
+### 3. Enable Kubeflow
+
+- Deploy Kubeflow with the command:
 
 ```
 microk8s.enable kubeflow
 ```
 
-The deployment process may take a few minutes. Once completed, the script will print out the port number and credentials to access the Kubeflow Dashboard.
-
+- The deployment process may take a few minutes. Once completed, the script will print out the port number and credentials to access the Kubeflow dashboard.
 
-## Access Kubeflow Dashboard
+## Access Kubeflow dashboard
 
 ### On you local host
 If you installed Microk8s on your local host, you simply need to open a web browser window and access the link given in the previous step. 
 
-### On Multipass or a Virtual Machine
+### On multipass or a virtual machine
 If running Microk8s on Multipass or a Virtual Machine, we need to create a SOCKS proxy. This can be done as follows:
```

Comment:
Did you apply the formatting—lower cases—here too? Lines 66 and 67 would look good if they're both consistent (`...on multipass or a virtual machine...`).